### PR TITLE
add retry for call to TimeRangeAnalysis SE API

### PR DESCRIPTION
**Pull request by pfrollo**

### CODE REVIEW

File: src/lib/actions/emailReport.js [hunk 1]
```diff
@@ -32,11 +38,29 @@ class EmailReport extends Action {
...
 
-    return request(requestOptions).catch((err) => {
-      console.error(`Email Report failed for ${this.options.fire_department__id}`);
-      console.dir(err);
+    console.log(`Calling TimeRangeAnalysis API for ${this.options.fire_department__id} fire department id`);
+
+    return promiseRetry(this.retryOptions, (retryCallback, attempt) => request(requestOptions).catch((error) => {
+      this.loggError(error, attempt);
+      retryCallback(error);
+    })).then(() => {
```
Comment (AelMalinka) [inline src/lib/actions/emailReport.js:46]: I think in order for subsequent calls to `.then` or `await` to get the request's response object it should be like
```js
.then((params) => {
  this.logSuccess();
  return params;
})
```

File: src/lib/actions/emailReport.js [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
 
 class EmailReport extends Action {
   constructor(options) {
     super(options);
+    this.retryOptions = {
+      retries: 10,
+      factor: 3,
+      minTimeout: 1000,
+      maxTimeout: 1000 * 60 * 5, // 5 minutes
+      randomize: false,
```
Comment (AelMalinka) [inline src/lib/actions/emailReport.js:16]: As I understand it these are triggered on a timer; might be good to change this to true so we don't have many retries happening at the same time; obviously if there is jitter in the scheduler already then fine as is but I'd think there isn't

File: src/lib/actions/emailReport.js (hunk 1 above)
Comment (pfrollo) [reply src/lib/actions/emailReport.js]: done

File: src/lib/actions/emailReport.js (hunk 2 above)
Comment (pfrollo) [reply src/lib/actions/emailReport.js]: done...it is a num value from 1 to 2 or false.

**Review by AelMalinka — APPROVED** [review]

**Pull request merged**